src/packedmonomial.jl

```diff
@@ -49,6 +49,27 @@ struct PackedMonomial{L,E,K} <: AbstractMonomial
     PackedMonomial{L,E}(bits::NTuple{K,UInt64}) where {L,E,K} = new{L,new_E(Val(E)),K}(bits)
     PackedMonomial{L,E,K}(bits::NTuple{K,UInt64}) where {L,E,K} = new{L,new_E(Val(E)),K}(bits)
 end
+nvariables(x::PackedMonomial{L}) where L = L
```

Comment:
This definition is not the same as the `nvariables` definition for `Monomial`.
`L` is the maximum number of supported variables.

Reply:
Yeah, I just added those quickly so I can use Groebner.jl :p it's not optimized.

Reply:
Also, computing nvariables for polynomial can be expensive. We have to take the union.